pricegraph/wasm/README.md

```diff
@@ -0,0 +1,30 @@
+# Pricegraph Wasm Bindings
+
+This crate provides WebAssembly bindings to the `pricegraph` crate so that it
+can be used from JS environments.
+
+## Usage
+
+```js
+const estimator = new PriceEstimator(encodedOrders);
+const [WETH, DAI] = [1, 7];
+console.log(estimator.estimatePrice(WETH, DAI, 100 * 10e18));
+esimator.free();
+```
```

Comment:
So is this from a node env? Do I have to import `PriceEstimator` or something?

Reply:
Added a bit more context, let me know if this is better.

Reply:
Nice thanks!